### PR TITLE
bump up crypki to v1.20.1 & remove elliptic.Unmarshal

### PR DESCRIPTION
**Pull request by py4chen**
PR updates crypki to have newer version of grpc and protobuf.
Since elliptic.Unmarshal has been deprecated since go 1.21.x, PR implements a parsing logic for EC public keys.

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by hkadakia — APPROVED** [review]
LGTM 👍🏼

File: attestation/yubiattest/parse.go [hunk 1]
```diff
@@ -668,20 +666,40 @@ func parsePublicKey(algo x509.PublicKeyAlgorithm, keyData *publicKeyInfo) (inter
...
-			Y:     y,
+		encodedPubKey := p.Bytes()
+		// The elliptic.Unmarshal() function has been deprecated in Go.
+		// As a workaround, we manually parse the public key by its index.
+		// For more details, see: https://github.com/golang/go/issues/63963
+		switch p.Curve() {
+		case ecdh.P256():
+			return &ecdsa.PublicKey{
+				Curve: elliptic.P256(),
+				X:     big.NewInt(0).SetBytes(encodedPubKey[1:33]),
```
Comment (maditya) [inline attestation/yubiattest/parse.go:685]: can we add a comment around not needing length check as `Bytes()` ensures the size is 133 https://github.com/golang/go/blob/release-branch.go1.23/src/crypto/ecdh/ecdh.go#L75-L80 ?

File: .github/workflows/golangci-lint.yml [hunk 2]
```diff
@@ -25,7 +25,7 @@ jobs:
     steps:
       - uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32 # v5.0.2
         with:
-          go-version: 1.19
+          go-version: 1.21
```
Comment (maditya) [inline .github/workflows/golangci-lint.yml:28]: nit: can we move to `go 1.23` here and below?

**Comment by py4chen** [comment]
Will address the comments in a follow-up PR.

**Pull request merged**